requirements.txt

```diff
@@ -23,6 +23,7 @@ django-redis==4.12.1
 django-statsd==2.5.2
 djangorestframework==3.11.0
 djangorestframework-csv==2.1.0
+freezegun==0.3.15
```

Comment:
don't thinik that's supposed to be in here?

Reply:
Hm, wouldn't be a problem but yeah, since only tests need freezegun and they're run with prod requirements.txt only, I just put it into the GH workflow.